target-platforms/packages/create-plaform/assets/build.js

```diff
@@ -84,9 +84,9 @@ function runBabel() {
 				'utf8'
 			);
 		}
-		catch (err) {
+		catch (error) {
```

Comment:
The GH action error above looks suspicious. You may need to rename this as `babelError` or something if this file is already using `error` as a variable in the file.

Reply:
Actually, the `error` variable declared before was not being used (and `yarn lint` was complaining), so I'm removing that one.

Reply:
Not really. It is used in line 89. The renaming is incorrect.

Reply:
ops, you are right